app/domain/authentication/authn_jwt/configuration_jwt_generic_vendor.rb

```diff
@@ -14,6 +16,10 @@ def initialize
         )
       end
 
+      def create_authentication_parameters(authentication_parameters)
+        @token_fetcher.call(authentication_parameters)
+      end
```

Comment:
this method is weird. it is called `create_authentication_parameters` but gets an `authentication_parameters` object as an input and calls the token_fetcher object. should it be renamed to `fetch_token_from_credentials`?

Reply:
The same there.
It will be addressed in next PRs